src/uct/ib/base/ib_md.c

```diff
@@ -1235,10 +1239,18 @@ ucs_status_t uct_ib_md_open_common(uct_ib_md_t *md,
     /* Check for GPU-direct support */
     md->reg_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
     if (md_config->enable_gpudirect_rdma != UCS_NO) {
-        /* check if GDR driver is loaded */
+        /* Check if GDR driver is loaded. Different driver/kernel configurations
+           can have different ways to check it */
```

Comment:
/* Check peer memory driver is loaded, different driver versions use different paths */

Reply:
```suggestion
        /* Check peer memory driver is loaded, different driver versions use different paths */
```